### PR TITLE
Various cleanup items, parameterization, ect

### PR DESCRIPTION
**Pull request by EdHurtig**
This is a fairly straightforward PR, it contains a bunch of changes to bring basic parameters back to working order such as the singularity port number, log file locations, etc..

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+default[:singularity][:database][:username] = 'singularity'
+default[:singularity][:database][:password] = '9thlevel'
+
+default[:singularity][:install_type] = 'package'
+
 default[:singularity][:log_level] = 'INFO'
 
 set[:mesos][:type] = 'mesosphere'
-set[:mesos][:mesosphere][:with_zookeeper] = true
+set[:mesos][:mesosphere]['with_zookeeper'] = true
```
Comment (jeffbyrnes) [inline attributes/default.rb]: Underscores should be fine for symbols, right?

Comment (eherot) [inline attributes/default.rb]: Yes.

File: recipes/package.rb [hunk 2]
```diff
@@ -3,6 +3,6 @@
 maven 'SingularityService' do
   group_id   'com.hubspot'
   classifier 'shaded'
-  version    node['singularity']['version']
+  version    node[:singularity][:version]
   dest       "#{node[:singularity][:home]}/bin/"
```
Comment (jeffbyrnes) [inline recipes/package.rb]: Seems like this should also restart Singularity, no?

**Comment by jeffbyrnes** [comment]
Great work :D

File: attributes/default.rb (hunk 1 above)
Comment (eherot) [inline attributes/default.rb]: although why the switch to strings here instead of symbols?

File: recipes/package.rb (hunk 2 above)
Comment (EdHurtig) [inline recipes/package.rb]: ah yeah it probably should :basketball:

File: attributes/default.rb (hunk 1 above)
Comment (EdHurtig) [inline attributes/default.rb]: oh derp, that was a mistake on my part.  I had initially started to standardize on strings  before talking with Jeff and deciding to standardize on symbols instead... This one got left behind : /

**Comment by EdHurtig** [comment]
@jeffbyrnes Fixed those things, going to test, merge now

Will release with your :+1:

**Comment by jeffbyrnes** [comment]
Thanks for just getting this done.

**Pull request merged**